.github/workflows/rust_ci.yml

```diff
@@ -39,39 +39,29 @@ jobs:
           command: check
           args: --locked
 
-      # Build & Run test & Collect Code coverage
-      - name: Run cargo-tarpaulin on ironfish-rust
-        uses: actions-rs/tarpaulin@v0.1
-        with:
-          version: "0.22.0"
-          args: --avoid-cfg-tarpaulin --manifest-path ironfish-rust/Cargo.toml --release -- --test-threads 1
-
-      # Upload code coverage to Codecov
-      - name: Upload to codecov.io
-        uses: codecov/codecov-action@v2.1.0
-        with:
-          token: ${{secrets.CODECOV_TOKEN}}
-          flags: ironfish-rust
+      # Note: ironfish-zkp is does not need this due to different licensing
+      - name: Check for license headers for ironfish-rust
+        run: ./ci/lintHeaders.sh ./ironfish-rust/src *.rs
 
-      - name: Check for license headers
+      - name: Check for license headers for ironfish-rust-nodejs
         run: ./ci/lintHeaders.sh ./ironfish-rust/src *.rs
 
       # fmt
       - uses: actions-rs/cargo@v1
         name: "`cargo fmt` check on ironfish-rust"
         with:
           command: fmt
-          args: --manifest-path ironfish-rust/Cargo.toml --all -- --check
+          args: --all -- --check
 
       # clippy
       - uses: actions-rs/cargo@v1
         name: "Clippy check on ironfish-rust"
         with:
           command: clippy
-          args: --manifest-path ironfish-rust/Cargo.toml --all-targets -- -D warnings
+          args: --all-targets -- -D warnings
 
-  ironfish_rust_nodejs:
-    name: ironfish-rust-nodejs
```

Comment:
the nodejs job only ran lint, so it was removed entirely. since it should only be napi glue code, it will have it's typescript tests tested in the typescript CI